.gitpod.yml

```diff
@@ -62,6 +62,8 @@ tasks:
       python3.9 /workspace/lila-db-seed/spamdb/spamdb.py
       redis-server --daemonize yes
       gp sync-done setup
+      ## additional vscode settings
+      cp /workspace/lila-gitpod/.vscode/settings.json /workspace/lila/.vscode/settings.json
```

Comment:
Getting an error on this line because `/workspace/lila/.vscode/` dir doesn't exist.

Maybe just

```
cp -r /workspace/lila-gitpod/.vscode /workspace/lila/.vscode
```